modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordSchemaFileProvider.kt

```diff
@@ -0,0 +1,34 @@
+package com.metalbear.mirrord
+import com.intellij.openapi.vfs.VirtualFile
+import com.jetbrains.jsonSchema.extension.JsonSchemaFileProvider
+import com.jetbrains.jsonSchema.extension.SchemaType
+import com.jetbrains.jsonSchema.impl.JsonSchemaVersion
+import com.jetbrains.jsonSchema.remote.JsonFileResolver
+
+class MirrordSchemaFileProvider: JsonSchemaFileProvider {
+
+    override fun isAvailable(vfile: VirtualFile): Boolean {
+        val path = vfile.path
+        return path.contains("mirrord") && path.endsWith(".json")
+    }
+
+    override fun getName(): String {
+        return "mirrord"
+    }
+
+    override fun getSchemaFile(): VirtualFile? {
+        return JsonFileResolver.urlToFile(this.remoteSource);
+    }
+
+    override fun getSchemaType(): SchemaType {
+        return SchemaType.remoteSchema
+    }
+
+    override fun getRemoteSource(): String {
+        return "https://raw.githubusercontent.com/metalbear-co/mirrord/main/mirrord-schema.json"
```

Comment:
We can also take the version from the local binary, there might be differences in the config. Especially since main != released, so kind of unstable. What do you think?

Reply:
That's a good point but the plugin does auto update and VSCode already fetches latest (via schema store) so I think - but taking your point and changing it to use latest release instead of just main :)